frontend/app/src/store/balances/manual.ts

```diff
@@ -89,9 +89,9 @@ export const useManualBalancesStore = defineStore('balances/manual', () => {
       .sort((a, b) => sortDesc(a.usdValue, b.usdValue));
   });
 
-  const getBreakdown = (asset: string): ComputedRef<AssetBreakdown[]> => computed<AssetBreakdown[]>(() => {
+  const getBreakdown = (asset: string, isLiability = false): ComputedRef<AssetBreakdown[]> => computed<AssetBreakdown[]>(() => {
```

Comment:
Also here same as below wrap internally and expose named methods so that it's clear on the consumer side.